src/Microsoft.PowerShell.ConsoleHost/resources/CommandLineParameterParserStrings.resx

```diff
@@ -210,4 +210,10 @@
   <data name="MissingWorkingDirectoryArgument" xml:space="preserve">
     <value>Cannot process the command because -WorkingDirectory requires an argument that is a directory path.</value>
   </data>
+  <data name="MTANotImplemented" xml:space="preserve">
+    <value>Parameter -MTA is not implemented on this platform.</value>
+  </data>
+  <data name="STANotImplemented" xml:space="preserve">
+    <value>Parameter -STA is not implemented on this platform.</value>
```

Comment:
Minor comment: we could add parameter with platform name.

Reply:
In the future, we may support other platforms like BSD.  I could change this to `non-Windows` if that helps.